[Split Build][no-commit] test ci using builder changes

PaliC opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #129012
* #129011



Test workflow: https://github.com/pytorch/pytorch/actions/runs/9601807487/job/26481169277

PaliC committed:
[Split Build][no-commit] test ci using builder changes

[ghstack-poisoned]

PaliC committed:
Update on "[Split Build][no-commit] test ci using builder changes"




[ghstack-poisoned]

PaliC committed:
Update on "[Split Build][no-commit] test ci using builder changes"


current test https://github.com/pytorch/pytorch/actions/runs/9573619911/job/26395380884


[ghstack-poisoned]

PaliC committed:
Update on "[Split Build][no-commit] test ci using builder changes"


current test https://github.com/pytorch/pytorch/actions/runs/9574045009


[ghstack-poisoned]

PaliC committed:
Update on "[Split Build][no-commit] test ci using builder changes"


current test https://github.com/pytorch/pytorch/actions/runs/9574045009


[ghstack-poisoned]

PaliC committed:
Update on "[Split Build][no-commit] test ci using builder changes"


current test https://github.com/pytorch/pytorch/actions/runs/9577458148


[ghstack-poisoned]

PaliC committed:
Update on "[Split Build][no-commit] test ci using builder changes"


current test https://github.com/pytorch/pytorch/actions/runs/9577458148


[ghstack-poisoned]

PaliC committed:
Update on "[Split Build][no-commit] test ci using builder changes"


current test https://github.com/pytorch/pytorch/actions/runs/9577458148


[ghstack-poisoned]

PaliC committed:
Update on "[Split Build][no-commit] test ci using builder changes"


current test https://github.com/pytorch/pytorch/actions/runs/9577458148


[ghstack-poisoned]

PaliC committed:
Update on "[Split Build][no-commit] test ci using builder changes"


current test https://github.com/pytorch/pytorch/actions/runs/9577458148


[ghstack-poisoned]

PaliC committed:
Update on "[Split Build][no-commit] test ci using builder changes"


current test https://github.com/pytorch/pytorch/actions/runs/9577458148


[ghstack-poisoned]

PaliC committed:
Update on "[Split Build][no-commit] test ci using builder changes"


current test https://github.com/pytorch/pytorch/actions/runs/9577458148


[ghstack-poisoned]

PaliC committed:
Update on "[Split Build][no-commit] test ci using builder changes"


Test workflow: https://github.com/pytorch/pytorch/actions/runs/9601807487/job/26481169277

[ghstack-poisoned]

atalman commented on .github/workflows/generated-linux-binary-manywheel-nightly.yml:
```diff
@@ -51,6 +51,7 @@ jobs:
       DOCKER_IMAGE: pytorch/manylinux-builder:cpu-main
       DESIRED_PYTHON: "3.8"
       build_name: manywheel-py3_8-cpu
+      use_split_build: True
```
Why do we splitting cpu builds ?

atalman commented on .github/scripts/generate_binary_build_matrix.py:
```diff
@@ -359,33 +359,10 @@ def generate_wheels_matrix(
             # 12.1 linux wheels require PYTORCH_EXTRA_INSTALL_REQUIREMENTS to install
             if (
-                arch_version in ["12.4", "12.1", "11.8"]
+                arch_version in ["12.4"]
```
Why do we need this change ?

PaliC committed:
Update on "[Split Build][no-commit] test ci using builder changes"




Test workflow: https://github.com/pytorch/pytorch/actions/runs/9601807487/job/26481169277

[ghstack-poisoned]

PaliC committed:
Update on "[Split Build][no-commit] test ci using builder changes"




Test workflow: https://github.com/pytorch/pytorch/actions/runs/9601807487/job/26481169277

[ghstack-poisoned]

PaliC committed:
Update on "[Split Build][no-commit] test ci using builder changes"




Test workflow: https://github.com/pytorch/pytorch/actions/runs/9601807487/job/26481169277

[ghstack-poisoned]

PaliC committed:
Update on "[Split Build][no-commit] test ci using builder changes"




Test workflow: https://github.com/pytorch/pytorch/actions/runs/9601807487/job/26481169277

[ghstack-poisoned]

PaliC committed:
Update on "[Split Build][no-commit] test ci using builder changes"




Test workflow: https://github.com/pytorch/pytorch/actions/runs/9601807487/job/26481169277

[ghstack-poisoned]
